dev-tools.toml

```diff
@@ -3,5 +3,6 @@ description = "Meetings for the dev tools team and its sub-teams"
 
 [meta]
 includes = [
+  "cargo.toml",
```

Comment:
Finally, support for `cargo.toml` (see Issue 45 at https://github.com/rust-lang/cargo/issues for context)

Reply:
We always have the option `kargo.toml`.